src/hints/tests.rs

```diff
@@ -4,30 +4,38 @@ mod tests {
     use cairo_vm::{
         serde::deserialize_program::ApTracking,
         types::exec_scope::ExecutionScopes,
-        utils::test_utils::*,
```

Comment:
This isn't accessible, I'm not sure why (it should be defined if `#[cfg(test)]` and is `pub` in that case)

Reply:
https://github.com/lambdaclass/cairo-vm/blob/0ea638f1227dccad33a73330cd070d4d68841ff7/vm/src/utils.rs#L86